utils/__init__.py

```diff
@@ -0,0 +1 @@
+import caas_utils
```

Comment:
Ditto this file. If you don't see any initialisation code being run in the `caas_utils`, then this file can be empty (the empty file just acting as a "you can import from this directory" marker).

Reply:
I think I understand it. This is quite tricky as I said previously.